src/sx/src/injectRuntimeStyles.js

```diff
@@ -82,16 +83,25 @@ export default function injectRuntimeStyles(styleBuffer: StyleBufferType) {
     if (node instanceof StyleCollectorNode) {
       if (hasStyleRule(matchFunction(node)) === false) {
         // apply missing styles
-        styleSheet.insertRule(node.printNodes().join(''), insertIndex);
+        const rule = serialize(
+          compile(node.printNodes({ trailingSemicolon: true }).join('')),
```

Comment:
Can't comment on that line, but you probably need this for line 62 as well, inject runtime keyframes.

Reply:
Done